lib/utils/index.js

```diff
@@ -611,6 +612,17 @@ class Utils {
 
     return false;
   }
+
+  static isAndroid(desiredCapabilities) {
+    const {avd} = desiredCapabilities;
+    const options = desiredCapabilities['goog:chromeOptions'] || desiredCapabilities['moz:firefoxOptions'];
+
+    if (avd && options && options.androidPackage) {
```

Comment:
`avd` is not a mandatory setting for running tests on Android.

Reply:
Done 👍